### DISCUSSION TITLE
Open source speech translate stacks

### DISCUSSION THREAD

**Discussion by ILG2021** [SYSTRAN/faster-whisper#130] [Ideas]
Nowadays I am using open source models to realize a speech to speech translator. Because I only have a 1070ti, I have to use ctranslator models. I use faster-whisper(really amazing fast) as the ASR, nllb-200-3.3b-ct2 as the text translator and gTTS for the tts. I found nllb-200 is not very precise so I have to change to deepl api. For the tts, I have tried conqui tts, their models are scattered and not easy to use, so I use gTTS directly. This is the stacks I used now. fast whisper + deepl api + gTTS

For the stacks, can anyone give me some suggestion? Thank you very much.

**Comment by phineas-pta** [comment]
you can take a look at this: https://github.com/ardha27/AI-Waifu-Vtuber
